crates/rome_js_formatter/src/js/expressions/arrow_function_expression.rs

```diff
@@ -316,9 +329,26 @@ fn should_break_chain(arrow: &JsArrowFunctionExpression) -> SyntaxResult<bool> {
                 AnyJsParameter::TsThisParameter(_) => false,
                 AnyJsParameter::JsRestParameter(_) => true,
             }),
-    };
+    }
+}
 
-    Ok(result)
+/// Returns `true` if the `arrow_parentheses` formatter option is enabled and parentheses can be safely avoided
```

Comment:
```suggestion
/// Returns `true` if the parentheses can be safely avoided.
```

The `arrow_parentheses` option is always passed :) maybe you meant to say something about its value?

Reply:
whoops! haha yeah